market_env/env.py

```diff
@@ -34,10 +37,24 @@ def __init__(self, user_id, env: ChainEnv, name: str):
         self.env = env
         self.env.users[name] = self
```

Comment:
Missing type hint everywhere @TammyQAQ 

what’s the point of having both “name” and “ID” as the user class’s key?